src/org/exist/backup/ConsistencyCheck.java

```diff
@@ -289,7 +289,7 @@ public Object start() {
                 try {
                     final ElementImpl root = (ElementImpl)doc.getDocumentElement();
                     if (root == null) {
-                        return new ErrorReport.ResourceError(ErrorReport.RESOURCE_ACCESS_FAILED, "Failed to access document data");
+                        return new ErrorReport.ResourceError(ErrorReport.RESOURCE_ACCESS_FAILED, "Failed to access data of document " + doc.getDocumentURI() );
```

Comment:
Nice. However, I think that `doc.getURI()` would be a safer choice than `doc.getDocumentURI()`. Could you change that please?